src/assert.js

```diff
@@ -119,19 +119,17 @@ Assert.prototype = {
 	},
 
 	equal: function( actual, expected, message ) {
-		/*jshint eqeqeq:false */
 		this.pushResult( {
-			result: expected == actual,
+			result: expected == actual,		// eslint-disable-line eqeqeq
```

Comment:
We can definitely do that, but there's another rule we would need to make a decision about.

Relevant configuration from eslint-config-jquery:
> 		"lines-around-comment": [
> 			"error",
> 			{
> 				"beforeLineComment": true
> 			}
> 		],

Basically, I would need to add a blank line before the comment. That could be okay, but I noticed the rest of the code was pretty condensed and didn't want to be adding a bunch of line breaks in the middle of objects without being sure it was okay. Using the inline form was more "convenient" in that regard because I could leave the existing structure.

How should I proceed?

* No changes (leave "eslint-disable-line" comments)
* Switch to eslint-disable-next-line and add necessary line breaks to conform with lines-around-comment
* Switch to eslint-disable-next-line and turn off the lines-around-comments rule in QUnit's ESLint configuration